configs/opendatasoft.json

```diff
@@ -13,7 +13,7 @@
         ]
       }
     },
-    "https://docs.opendatasoft.com/"
+    "https://help.opendatasoft.com/"
```

Comment:
This URL is too broad considering your current set of selectors. Please remove it or update your selectors.
Please note that DocSearch is only provided for documentation

Reply:
Please provide more details if you want us to encompass more pages

Reply:
Right, I removed this URL, the other one is enough.
I have updated the branch.